trino/trino.go

```diff
@@ -214,6 +219,20 @@ func (c *Config) FormatDSN() (string, error) {
 			credkv = append(credkv, k+mapKeySeparator+v)
 		}
 	}
+
+	var roles []string
+	if c.Roles != nil {
+		if v, ok := c.Roles.(string); ok {
+			roles = append(roles, sistemRole+mapRolesSeparator+fmt.Sprintf("ROLE{%q}", v))
+		} else if v, ok := c.Roles.(map[string]string); ok {
+			for k, v := range v {
+				roles = append(roles, k+mapRolesSeparator+fmt.Sprintf("ROLE{%q}", v))
```

Comment:
Don't mix string concatenation and format strings. We also don't really need constants for the system role and the separator here, if this is the only place where they're used.
```suggestion
			roles = append(roles, fmt.Sprintf("system=ROLE{%q}", v))
		} else if v, ok := c.Roles.(map[string]string); ok {
			for k, v := range v {
				roles = append(roles, fmt.Sprintf("%s=ROLE{%q}", k, v))
```